README.md

```diff
@@ -45,9 +46,7 @@ import * as PXBThemes from '@pxblue/react-native-themes';
 
 ### Dark Theme
 
-The theming mechanism for React Native Paper is a bit limited in the amount of customization available for components. Because of this, there are two dark themes available from PX Blue that should be applied to different components.
-
-The main theme should be applied using a `Provider` that wraps your application and passing in the theme (`blueDark`). This will be applied to the majority of the component in the RNP library.
+To use the light theme in your application, simply wrap the app in a `Provider` and pass in your desired theme (`blueDark`).
```

Comment:
Since this instruction is the same for both light and dark now, I would pull it up into the preceding paragraph and then have the dark/light subsections just show the code snippets.

Reply:
And for dark theme, mention that users should use the wrapper components and reference the section below.